promql/functions.go

```diff
@@ -14,7 +14,9 @@
 package promql
 
 import (
+	"fmt"
```

Comment:
Minor thing, when I was playing around with this code, the compiler warned that this import wasn't actually being used at all.